File: ios/Classes/channels/methods/MediaChannelHandler.swift
//
//  MediaChannelHandler.swift
//  Runner
//
//  Created by bryosabian on 22/11/2023.
//

import Foundation


class MediaChannelHandler : AbstractMethodChannelHandler {

    override func register() {
        register("takePicture", PhotoMethodChannelHandler())
        register("choosePicture", GalleryMethodChannelHandler())
    }
}

File: sabian_native_common_android/android/src/main/kotlin/com/sabiantech/sabian_native_common_android/channels/methods/MediaChannelHandler.kt
package com.sabiantech.sabian_native_common_android.channels.methods


import com.sabiantech.sabian_native_common_android.Config
import com.sabiantech.sabian_native_common_android.channels.methods.photo.GalleryMethodChannelHandler
import com.sabiantech.sabian_native_common_android.channels.methods.photo.PhotoMethodChannelHandler


class MediaChannelHandler : AbstractMethodChannelHandler() {
    companion object {
        const val MEDIA_ERROR_CODE = Config.MEDIA_ERROR_CODE
    }

    override fun register() {
        register("takePicture", PhotoMethodChannelHandler())
        register("choosePicture", GalleryMethodChannelHandler())
    }

}